code/modules/research/designs/designs.dm

```diff
@@ -62,4 +62,4 @@ other types of metals and chemistry for reagents).
 	return new build_path(newloc)
 
 /datum/design/item
-	build_type = PROTOLATHE
\ No newline at end of file
+	build_type = PROTOLATHE
```

Comment:
If there are no changes to the file, you should remove it from your PR.

Reply:
Thanks, missed that. It's surprisingly hard to convince any of my editors to deliberately remove the newline though... Ended up using DreamMaker. x.x